### PR TITLE
[FIX] flake8 (import in bad place);

### PR DESCRIPTION
**Pull request by legalsylvain**
Fix some "E402 module level import not at top of file"

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1:

**Comment by moylop260** [comment]
:+1:

**Comment by legalsylvain** [comment]
Hi, thanks for your review. I don't understand why runbot fail for this PR. Anybody has an idea ?

**Comment by pedrobaeza** [comment]
There are 2 warnings in runbot:

```
2016-01-30 13:12:23,194 9423 WARNING 3140686-346-751d47-all root: Sentry DSN not defined in config file
2016-01-30 13:12:24,315 9423 WARNING 3140686-346-751d47-all openerp.modules.module: 
Ambiguous import: the OpenERP module `resource` is shadowed by another
module (available at /srv/openerp/instances/openerp-oca-runbot/sandbox/lib/python2.7/lib-dynload/resource.x86_64-linux-gnu.so).
To import it, use `import openerp.addons.<module>.`.
```

**Comment by legalsylvain** [comment]
Oh. I see. the flake8 fix I wrote breaks runbot because of a "ugly hack" : 
https://github.com/OCA/server-tools/pull/346/files#diff-3d591500c249b214c340e905d5ae6290R47

Do you have an idea how to fix that ?
CC : @gurneyalex 

regards.

**Comment by gurneyalex** [comment]
@legalsylvain I just found PEP328... 

Can you add this before the first import:

```
from __future__ import absolute_import
```

Then the `import resource` statement should no longer cause a warning.

**Comment by StefanRijnhart** [comment]
You can now remove the ugly hack ;-)

File: server_monitoring/models/server_monitor_process.py [hunk 1]
```diff
@@ -60,6 +53,13 @@
 from openerp import SUPERUSER_ID
 from openerp.tools import DEFAULT_SERVER_DATETIME_FORMAT
 
+# ugly hack to avoid a WARNING message when importing stdlib resource module
+_logger = logging.getLogger('openerp.modules.module')
+_saved_log_level = _logger.getEffectiveLevel()
+_logger.setLevel(logging.ERROR)
+
+_logger.setLevel(_saved_log_level)
```
Comment (dreispt) [inline server_monitoring/models/server_monitor_process.py]: This is not doing anything, isn't it?

Comment (gurneyalex) [inline server_monitoring/models/server_monitor_process.py]: yep it should be removed.

**Comment by dreispt** [comment]
:+1:

**Comment by sebalix** [comment]
:+1:

**Pull request merged**